### DISCUSSION TITLE
Add new messages to existing discussions in the default folder

### DISCUSSION THREAD

**Discussion by Gilbert535** [ReagentX/imessage-exporter#252] [Ideas]
I am trying to use iMessage-exporter as a backup tool with a view to only keep one year of iMessages in iCloud but have all past messages available  in case I ever need them.

If I export iMessages today, is there a way, say in a month to append to today's export all new messages sent and received during coming month in each respective conversation? That way , I would always have an up to date back up of all iMessages.

I did try to export iMessages to the default folder and then export again and got the message that the folder "contains existing "html" export data!"

I understand  I could export iMessages using from and to dates and create a separate folder for each time period, but It would be more convenient to have all messages under the same folder.

**Comment by ReagentX** [comment]
No, there isn't a way to do this. Exports are designed to be run once.
